### PR TITLE
[Chronos] Solve the problem of slow initialization for tf2 seq2seq forecaster

### PR DESCRIPTION
**Pull request by devWangBin**
### 1. Why the change?
Solve the problem of slow initialization for tf2 seq2seq forecaster.

### 2. User API changes
Add new API:
bigdl.chronos.model.tf2.Seq2Seq_keras: **model_creator_auto**

### CODE REVIEW

File: python/chronos/src/bigdl/chronos/model/tf2/Seq2Seq_keras.py [hunk 1]
```diff
@@ -143,6 +143,22 @@ def from_config(cls, config):
...
+                        teacher_forcing=config.get("teacher_forcing", False))
+    learning_rate = config.get('lr', 1e-3)
+    model.compile(optimizer=getattr(tf.keras.optimizers,
+                                    config.get("optim", "Adam"))(learning_rate),
+                  loss=config.get("loss", "mse"),
+                  metrics=[config.get("metics", "mse")])
+    return model
+
+
+def model_creator_auto(config):
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/model/tf2/Seq2Seq_keras.py:161]: add comment here to state why we need this special model creator.

Comment (devWangBin) [reply python/chronos/src/bigdl/chronos/model/tf2/Seq2Seq_keras.py:161]: The reason is as follows:
The new **model_creator_auto** function adds two lines of code compared to the original **model_creator**, which will run (model(inputs)) to initialize the weights when creating the model, which is required for the AutoSeq2Seq model.

Comment (TheaperDeng) [reply python/chronos/src/bigdl/chronos/model/tf2/Seq2Seq_keras.py:161]: yeah just put a simple version to the src's comment. e.g. add model(inputs) in this model creator to initialize the weights

Comment (devWangBin) [reply python/chronos/src/bigdl/chronos/model/tf2/Seq2Seq_keras.py:161]: got it

Comment (TheaperDeng) [reply python/chronos/src/bigdl/chronos/model/tf2/Seq2Seq_keras.py:161]: great, will merge once the pr validations pass

**Pull request merged**